src/App.tsx

```diff
@@ -1,24 +1,327 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
```

Comment:
App file is to large. ill recomend to create context or take some code in seperate components